### ISSUE TITLE
ALSA Error when invoked first time but OK second time 

### ISSUE THREAD

**Issue by AL1-1956** [mido/mido#542]
Hello everybody
First of all I am French and I am 67 yo so my English is a little rusty....

I have a strange behaviour with mido and i put the thing here
=====================================================================
>>> import mido

First Time sending the command
>>> mido.get_output_names()
Traceback (most recent call last):
  File "<input>", line 1, in <module>
    mido.get_output_names()
  File "/home/al1/.local/lib/python3.10/site-packages/mido/backends/backend.py", line 179, in get_o
utput_names
    devices = self._get_devices(**self._add_api(kwargs))
  File "/home/al1/.local/lib/python3.10/site-packages/mido/backends/backend.py", line 167, in _get_
devices
    return self.module.get_devices(**self._add_api(kwargs))
  File "/home/al1/.local/lib/python3.10/site-packages/mido/backends/rtmidi.py", line 55, in get_dev
ices
    mi = rtmidi.MidiIn(rtapi=rtapi)
  File "_rtmidi.pyx", line 820, in _rtmidi.MidiIn.__cinit__
_rtmidi.SystemError: MidiInAlsa::initialize: error creating ALSA sequencer client object.

Second time sending the command

>>> mido.get_output_names()
['Midi Through:Midi Through Port-0 14:0']

So basically the first time i send the command "mido.get_output_names()" , get an error
but if I repeat the command It's OK .....
It is Python 3.10.12 in Lubuntu

**Comment by rdoursenaud** [comment]
Hi,

Thanks for taking the time to report.

This is a known issue from our dependency SpotlightKid/python-rtmidi#138 already reported in #479.

The fact that the command goes through the second time is new information I believe.

There's unfortunately nothing we can do from our end.

Please see these for a workaround and subscribe if you want to track their proper resolution.
